site/content/3.11/develop/integrations/spring-data-arangodb/_index.md

```diff
@@ -37,66 +39,131 @@ Spring Data ArangoDB is compatible with:
 
 {{< /tabs >}}
 
-## Maven
+## Get started
 
-To use Spring Data ArangoDB in your project, your build automation tool needs to
-be configured to include and use the Spring Data ArangoDB dependency.
-Example with Maven (substitute `x.x.x` with the latest Spring Data ArangoDB version):
+This tutorial is about how to configure [Spring Data ArangoDB](https://github.com/arangodb/spring-data)
+without using Spring Boot Starter ArangoDB.
 
-```xml
-<dependency>
-  <groupId>com.arangodb</groupId>
-  <artifactId>arangodb-spring-data</artifactId>
-  <version>x.x.x</version>
-</dependency>
-```
+For a more extensive tutorial about the features of Spring Data ArangoDB and
+Spring Boot support, see the [Spring Boot Starter](../spring-boot-arangodb.md)
+documentation.
 
-There is a [demonstration app](https://github.com/arangodb/spring-data-demo), which contains common use cases and examples of how to use Spring Data ArangoDB's functionality.
+### Build a project with Maven
 
-## Configuration
+Set up a project and add every needed dependency. This demo uses Maven and
+Spring Boot.
 
-{{< tabs "spring-data" >}}
+Create a Maven `pom.xml`:
 
-{{< tab "Version 4" >}}
-You can use Java to configure your Spring Data environment as show below.
-Setting up the underlying driver (`ArangoDB.Builder`) with default configuration
-automatically loads a properties file `arangodb.properties`, if it exists in the
-classpath.
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <relativePath/>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.3.4</version>
+    </parent>
+
+    <groupId>com.arangodb</groupId>
+    <artifactId>spring-data-arangodb-tutorial</artifactId>
+    <version>1.0.0</version>
+
+    <name>spring-data-arangodb-tutorial</name>
+    <description>ArangoDB Spring Data Tutorial</description>
+
+    <properties>
+        <java.version>21</java.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.arangodb</groupId>
+            <artifactId>arangodb-spring-data</artifactId>
+            <version>4.4.2</version> <!-- Substitute with the latest Spring Data ArangoDB version -->
```

Comment:
If we say to use the latest version here, I suppose we should also do that above for the other deps?

Reply:
Yes, but note that:
- Spring Data ArangoDB 4.*.* is compatible with all the still supported Spring Boot 3.*.* (see https://docs.arangodb.com/stable/develop/integrations/spring-data-arangodb/#supported-versions)
- Spring Boot Starter ArangoDB 3.x-* is compatible with Spring Boot 3.x.* (see https://docs.arangodb.com/stable/develop/integrations/spring-boot-arangodb/#supported-versions)